stablehlo/dialect/VhloOps.td

```diff
@@ -240,6 +240,15 @@ def VHLO_ClzOpV1 : VHLO_Op<"count_leading_zeros_v1", "0.9.0", "current"> {
   let results = (outs VHLO_AnyType:$result);
 }
 
+def VHLO_CollectiveBroadcastOpV1 : VHLO_Op<"collective_broadcast_v1", "0.9.0", "current"> {
```

Comment:
"0.9.0" is supposed to be the earliest release an op can be used in, "current" is the latest release an op was used in.

In this case should be `"0.16.0", "current"`